mobile/package.json

```diff
@@ -40,7 +45,8 @@
     "android": "expo start --android",
     "ios": "expo start --ios",
     "web": "expo web",
-    "eject": "expo eject"
+    "eject": "expo eject",
+    "dev": "unset BASE_URL && npm start"
```

Comment:
I'd rather this be more explicit, as `"dev": "BASE_URL=http://localhost:8080 npm start"`